Resources/Prototypes/_SpaceCats/Actions/cloak.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  id: ADTActionToggleCloak
+  categories:
+    - HideSpawnMenu
+  name: Toggle Cloak
+  description: Enable or Disable your Thermal Cloak
+  components:
+    - type: InstantAction
+      event: !type:ThermalCloakTurnInvisibleActionEvent
+      useDelay: 5
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)